UPGRADE.md

```diff
@@ -994,14 +997,16 @@ export const UserCreate = (props) => (
             <TextInput source="lastName" />
             <BooleanInput source="hasEmail" label="Has email ?" />
 -           <DependentInput dependsOn="hasEmail">
-+           <DependsOn source="hasEmail">
```

Comment:
don't trust GitHub's diff, there is no trace of `<DependsOn>` in the last version.